test/unit/org/apache/cassandra/cql3/validation/operations/AlterTest.java

```diff
@@ -568,6 +567,105 @@ public void testDoubleWith() throws Throwable
         }
     }
 
+
+    @Test
+    public void testAlterTableWithMemtable() throws Throwable
+    {
+        createTable("CREATE TABLE %s (a text, b int, c int, primary key (a, b))");
+        assertSame(MemtableParams.DEFAULT.factory, getCurrentColumnFamilyStore().metadata().params.memtable.factory);
+
+        assertRows(execute(format("SELECT memtable FROM %s.%s WHERE keyspace_name = ? and table_name = ?;",
+                                  SchemaConstants.SCHEMA_KEYSPACE_NAME,
+                                  SchemaKeyspaceTables.TABLES),
+                           KEYSPACE,
+                           currentTable()),
+                   row(map()));
```

Comment:
This block appears six times across the test, only changing the `map` arguments. We could reduce duplication with an utility method like:
```java
private void assertMemtableOptions(Object... options) throws Throwable
{
    assertRows(execute(format("SELECT memtable FROM %s.%s WHERE keyspace_name = ? and table_name = ?",
                              SchemaConstants.SCHEMA_KEYSPACE_NAME,
                              SchemaKeyspaceTables.TABLES),
                       KEYSPACE,
                       currentTable()),
               row(map(options)));
}
```

Reply:
Done